src/main/java/billennium/tests/model/QuestionModel.java

```diff
@@ -6,7 +6,11 @@
 import java.util.List;
 
 @Builder
-@Value
+@Value // TODO UWAGA do wszystkich klas w pakiecie model  doczytaj co robi adnotacja Value, dzieki niej nie jest
+// potrzebny kwalifikator dostepu dla pol w klasie
+// TODO kolejna uwaga do wszystkiego w tym pakiecie: Czy nazwanie tego Model jest odpowiednie ? Dlaczego te klasy sa
+//  tak nazwane. Mi to bardziej wyglada na jakieś projekcje a nie Model. Tutaj wazne jest nazewnictwo w projekcie. Ja
+//  zawsze spotyjalem sie na backendzie ze Model == Entity. Moze to jest jakis DTO ? Proszę o wytlumaczenie dlaczego tak
```

Comment:
w starym projekcie po prostu używaliśmy Model zamiast DTO, akurat Model == Entity nie spotkałem się

Reply:
A cóż co do @Value to ok  ale dla mnie to źle wygląda jak nie ma w klasie tego modyfikatora private mimo że jest on default w takim przypadku

Reply:
Do zostananowienia się: czy Model w typ  przypadku nie jest projekcją. 
co do value to nie ma żadnego znaczenia, że wygląda to dziwnie według Ciebie. Po pierwsze trzeba używać tą adnotacje świadomie i wiedzieć że ona ustawia pola na private. Po drugie jeżeli kod z @Value i private na polach zostanie przepuszczony przez Sonara czy inne narzędzie tego typu wyjdą błędy a jak wiadomo nie chcemy mieć błędów w naszym wyimaginowany sonarze.